database_apis/users_database.py

```diff
@@ -117,12 +119,44 @@ def logout(self):
         
     def get_current_user(self):
         return self.user
+    
```

Comment:
The changes here are making the function in the user_database related to the auto login function for the login file.